utility/animation/animation.sh

```diff
@@ -0,0 +1,250 @@
+
+set -e
+
+typingspeed=10
+shortpause=1
+mediumpause=2
+longpause=5
```

Comment:
🤔: If we pass `--idle-time-limit 2.5` to asciinema, then is `$longpause` limited to 2.5s?